rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/400_geo_bounding_box_query.yml

```diff
@@ -0,0 +1,119 @@
+---
+setup:
+  - do:
+      indices.create:
+        index: test_index
+        body:
+          mappings:
+            properties:
+              location:
+                type: geo_point
+
+  - do:
+      bulk:
+        refresh: true
+        body:
+          - '{"index": {"_index": "test_index", "_id": "u1"}}'
+          - '{"location": "45.0, 45.0"}'
+          - '{"index": {"_index": "test_index", "_id": "u2"}}'
+          - '{"location": {"lat" : -45.0 , "lon" : 45.0}}'
+          - '{"index": {"_index": "test_index", "_id": "u3"}}'
+          - '{"location": [-45.0, 45.0]}'
+          - '{"index": {"_index": "test_index", "_id": "u4"}}'
+          - '{"location": "POINT(-45.0 -45.0)"}'
```

Comment:
How about a GeoJSON example? It's not particularly important, but I do notice that our decision to try focus on GeoJSON and WKT (and soft deprecate the older string format) is not well represented in our test cases.